jenkins-base/templates/cluster-secrets-cleanup.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "hookname" . }}
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+  annotations:
+    "helm.sh/hook": pre-delete
+spec:
+  completions: 1
+  parallelism: 1
+  template:
+    metadata:
+      name: {{ template "hookname" . }}
+      labels:
+        heritage: {{.Release.Service | quote }}
+        release: {{.Release.Name | quote }}
+        chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    spec:
+     restartPolicy: "Never"
+     containers:
+      - name: {{ template "hookname" . }}
+        image: google/cloud-sdk 
+        command: 
+          - "/bin/bash"
+          - "-cx"
+          - |
+            # delete pre install hook secrets
```

When would these fail?  And why do we swallow the error?